src/pages/services.js

```diff
@@ -0,0 +1,157 @@
+              buttonContent="Discover More"
+              class="button-green"
+              handleClick={handleClick}
+            />
+          </div>
+          <div className="service-box">
+            <div className="icon">
+              <img src={serviceIcon03} alt=""></img>
+            </div>
+            <h5>Third Box Service</h5>
+            <p>
+              Quisque finibus libero augue, in ultrices quam dictum id. Aliquam
+              quis tellus sit amet urna tincidunt bibendum.
+            </p>
+            <Button
+              buttonContent="More Detail"
+              class="button-green"
+              handleClick={handleClick}
+            />
+          </div>
+          <div className="service-box">
+            <div className="icon">
+              <img src={serviceIcon01} alt=""></img>
+            </div>
+            <h5>Fourth Box Service</h5>
+            <p>
+              Fusce sollicitudin feugiat risus, tempus faucibus arcu blandit
+              nec. Duis auctor dolor eu scelerisque vestibulum.
+            </p>
+            <Button
+              buttonContent="Read More"
+              class="button-green"
+              handleClick={handleClick}
+            />
+          </div>
+          <div className="service-box">
+            <div className="icon">
+              <img src={serviceIcon01} alt=""></img>
+            </div>
+            <h5>Fifth Box Service</h5>
+            <p>
+              Fusce sollicitudin feugiat risus, tempus faucibus arcu blandit
+              nec. Duis auctor dolor eu scelerisque vestibulum.
+            </p>
+            <Button
+              buttonContent="Discover"
+              class="button-green"
+              handleClick={handleClick}
+            />
+          </div>
+          <div className="service-box">
+            <div className="icon">
+              <img src={serviceIcon02} alt=""></img>
+            </div>
+            <h5>Sixth Box Service</h5>
+            <p>
+              Ut nibh velit, aliquam vitae pellentesque nec, convallis vitae
+              lacus. Aliquam porttitor urna ut pellentesque.
+            </p>
+            <Button
+              buttonContent="Detail"
+              class="button-green"
+              handleClick={handleClick}
+            />
+          </div>
+          <div className="service-box">
+            <div className="icon">
+              <img src={serviceIcon03} alt=""></img>
+            </div>
+            <h5>Seventh Box Service</h5>
+            <p>
+              Sed a consequat velit. Morbi lectus sapien, vestibulum et sapien
+              sit amet, ultrices malesuada odio. Donec non quam.
+            </p>
+            <Button
+              buttonContent="Read More"
+              class="button-green"
+              handleClick={handleClick}
+            />
+          </div>
+        </Carousel>
+      </div>
+    </div>
+  );
+}
+
+function DotCarousel(props) {
+  const classBackFrame = props.active
```

Comment:
I would use `clsx` module for this kind of situations
https://www.npmjs.com/package/clsx

Reply:
fixed...